core/lib/zksync_core/src/api_server/web3/namespaces/eth.rs

```diff
@@ -229,6 +229,9 @@ impl EthNamespace {
         const METHOD_NAME: &str = "get_filter_logs";
 
         let method_latency = API_METRICS.start_call(METHOD_NAME);
+        if !self.state.api_config.filters_enabled {
```

Comment:
This works, but IMO it'd be more idiomatic if `RpcState.installed_filters` would be made `Option`al and set to `None` when filters are disabled. If this was the case, you couldn't accidentally miss filter uses even if you wanted.

Reply:
Very good point, fixed in https://github.com/matter-labs/zksync-era/pull/1078/commits/71bb815d592c8c119cd0ad568bba4a60548ca885